features/facebookImport/src/model/analyses/ministories/connected-advertisers-analysis.js

```diff
@@ -3,12 +3,21 @@ import analysisKeys from "../utils/analysisKeys";
 
 export default class ConnectedAdvertisersAnalysis extends RootAnalysis {
     async analyze({ dataAccount }) {
-        if (dataAccount.connectedAdvertisers.length === 0) return;
-        dataAccount.analyses[analysisKeys.connectedAdvertisersCount] =
-            dataAccount.connectedAdvertisers.length;
-        dataAccount.analyses[analysisKeys.connectedAdvertiserNames] =
-            dataAccount.connectedAdvertisers.map(
-                (connectedAdvertiser) => connectedAdvertiser.name
-            );
+        if (dataAccount.connectedAdvertisers?.length > 0) {
+            dataAccount.analyses[analysisKeys.connectedAdvertisersCount] =
+                dataAccount.connectedAdvertisers.length;
+            dataAccount.analyses[analysisKeys.connectedAdvertiserNames] =
+                dataAccount.connectedAdvertisers.map(
+                    (connectedAdvertiser) => connectedAdvertiser.name
+                );
+        }
+        if (dataAccount.connectedAdvertisersAllTypes?.length > 0) {
+            dataAccount.analyses[analysisKeys.connectedAdvertisersCount] =
+                dataAccount.connectedAdvertisersAllTypes.length;
```

Comment:
So if `connectedAdvertisersAllTypes` is present, that overwrites `connectedAdvertisers` - that's the desired behaviour, right?